### PR TITLE
Add docs to mention 1.26 support

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*
Add docs to mention 1.26 support

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/clusterspec/cloudstack.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ Number of etcd members
 Refers to the Kubernetes object with CloudStack specific configuration for your etcd members. See `CloudStackMachineConfig Fields` below.
 
 ### kubernetesVersion (required)
-The Kubernetes version you want to use for your cluster. Supported values: `1.24`, `1.23`, `1.22`, `1.21`
+The Kubernetes version you want to use for your cluster. Supported values: `1.24`, `1.23`, `1.22`
```
Comment (drewvanstone) [inline docs/content/en/docs/reference/clusterspec/cloudstack.md:239]: Missing 1.25 and 1.26?

Comment (taneyland) [reply docs/content/en/docs/reference/clusterspec/cloudstack.md:239]: We don't support 1.25 and 1.26 on CloudStack yet

**Pull request merged**